pynetworktables2js/js/tableviewer.js

```diff
@@ -0,0 +1,290 @@
+"use strict";
+
+(function() {
+
+	if ($ === undefined) {
+	    alert("jQuery must be downloaded+included to use tableviewer.js!");
+	    return;
+	}
+
+	function Tableviewer($el) {
+		this.$el = $el.addClass('tableviewer');
+		this.ntRoot = {
```

Comment:
Are you trying to use this as a map? What about NT keys with weird symbols in them? That's why I use the d3_map in the main NT websocket class...